Moya-ModelMapper.podspec

```
@@ -35,21 +35,21 @@ Pod::Spec.new do |s|
 
   s.subspec "Core" do |ss|
     ss.source_files  = "Sources/Moya-ModelMapper/*.swift"
-    ss.dependency "Moya", "14.0.0-beta.5"
+    ss.dependency "Moya", "14.0.0"
```

this would mean we would only accept Moya 14.0.0 (e.g. Moya 14.0.1 or 14.1 if it comes would need additional upgrade to the Moya-ModelMapper as well), so let's just do:

```suggestion
    ss.dependency "Moya", "~> 14.0"
```